mimecast/assets/dashboards/mimecast_rejection.json

```diff
@@ -0,0 +1,627 @@
+{
+    "title": "Mimecast - Rejection",
+    "description": "This dashboard provides information about the Rejections generated on Mimecast Email Security.",
+    "widgets": [
+        {
+            "id": 5527090904543100,
+            "definition": {
+                "title": "",
+                "banner_img": "https://getlogovector.com/wp-content/uploads/2020/09/mimecast-logo-vector.png",
+                "show_title": true,
+                "type": "group",
+                "layout_type": "ordered",
+                "widgets": [
+                    {
+                        "id": 911775482073612,
+                        "definition": {
+                            "type": "note",
+                            "content": "Gain better visibility into your organization's security events by monitoring Mimecast Rejection logs with this dashboard. \n\nThis dashboard provides a centralized view of rejected messages, enabling administrators to monitor and analyze email traffic that has been blocked due to security concerns.\n\nFor more information, see the [Mimecast Email Security Integration Documentation](https://docs.datadoghq.com/integrations/mimecast).\n\n**Tips**\n- Use the timeframe selector in the top right of the dashboard to change the default timeframe.\n- Clone this dashboard to rearrange, modify and add widgets and visualizations.",
```

Comment:
After this sentence, please add a sentence describing what Mimecast Rejection logs includes.
`Gain better visibility into your organization's security events by monitoring Mimecast Rejection logs with this dashboard. [insert new sentence]`

Reply:
We have added the details. 👍